board/safety/safety_subaru.h

```diff
@@ -216,6 +230,15 @@ static int subaru_tx_hook(CANPacket_t *to_send) {
     violation |= steer_torque_cmd_checks(desired_torque, -1, limits);
   }
 
+  if (addr == MSG_SUBARU_ES_LKAS_ANGLE) {
+    int desired_angle = (GET_BYTES(to_send, 5, 3) & 0x1FFFFU);
+    desired_angle = to_signed(desired_angle, 17);
+    bool lkas_request = GET_BIT(to_send, 12U);
+
+    const SteeringLimits limits = SUBARU_ANGLE_STEERING_LIMITS;
+    violation |= steer_angle_cmd_checks(desired_angle, lkas_request, limits);
```

Comment:
does the command angle from the stock system go to 0 when disabled, or matches the meas angle, and is it limited to a certain degree? On Toyota LTA for example, it matched meas, stopping at 95 degrees. Anything higher and it faulted the EPS

Reply:
IIRC it follows the measured steering angle, at least to 300 deg. I'll double check that though

Reply:
It tracks the angle at least between -433 and 395, I will try and get a route with lock to lock from someone

f3b34c0d2632aa83|2023-07-23--18-57-25
![image](https://github.com/commaai/panda/assets/9648890/874bb361-f241-4252-870b-48963dd414fc)